src/components/organisms/AssetActions/Trade/Output.tsx

```diff
@@ -29,8 +34,12 @@ export default function Output({
       setSwapFee(`${Number(swapFee) * 100}`)
```

Comment:
is there a reason why this was not converted to decimal?

Reply:
I missed this one, thanks for catching it! I'll go through all files again to check there aren't any missing (and needed) conversations like this one.